Description: Make some .desktop files standard compliant
Author: Nicolas Bourdaud <nicolas.bourdaud@gmail.com>
Origin: vendor
Forwarded: https://github.com/linuxmint/Cinnamon/pull/987
Applied-Upstream: no
Last-Update: 2012-07-29

--- a/data/cinnamon.desktop.in.in
+++ b/data/cinnamon.desktop.in.in
@@ -7,7 +7,7 @@
 X-GNOME-Bugzilla-Product=cinnamon
 X-GNOME-Bugzilla-Component=general
 X-GNOME-Bugzilla-Version=@VERSION@
-Categories=GNOME;GTK;Core;
+Categories=GNOME;GTK;System;Core;
 OnlyShowIn=GNOME;
 NoDisplay=true
 X-GNOME-Autostart-Phase=WindowManager
--- a/files/usr/share/applications/cinnamon-settings.desktop
+++ b/files/usr/share/applications/cinnamon-settings.desktop
@@ -5,6 +5,5 @@
 Icon=preferences-system
 Terminal=false
 Type=Application
-Encoding=UTF-8
 Categories=GNOME;GTK;Settings;DesktopSettings;
 StartupNotify=false
--- a/files/usr/share/applications/cinnamon2d.desktop
+++ b/files/usr/share/applications/cinnamon2d.desktop
@@ -7,7 +7,7 @@
 X-GNOME-Bugzilla-Product=cinnamon
 X-GNOME-Bugzilla-Component=general
 X-GNOME-Bugzilla-Version=1.5.2
-Categories=GNOME;GTK;Core;
+Categories=GNOME;GTK;System;Core;
 OnlyShowIn=GNOME;
 NoDisplay=true
 X-GNOME-Autostart-Phase=WindowManager
